<?xml version="1.0"?>
<launch>

    <node pkg="racecar_rl_environments" type="reward_server.py" name="reward_server" output="screen">
    </node>

    <node pkg="racecar_rl_environments" type="reward_client.py" name="reward_client" output="screen">
    </node>


</launch>
